crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -816,6 +816,42 @@ pub struct CybersourceErrorInformation {
     message: Option<String>,
 }
 
+impl<F, T>
+    TryFrom<(
+        &CybersourceErrorInformationResponse,
+        types::ResponseRouterData<F, CybersourcePaymentsResponse, T, types::PaymentsResponseData>,
+    )> for types::RouterData<F, T, types::PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
```

Comment:
This can be From impl instead of TryFrom

Reply:
Done